api/src/services/users.js

```diff
@@ -34,7 +34,6 @@ const RESTRICTED_SETTINGS_EDITABLE_FIELDS = [
   'fullname',
   'email',
   'phone',
-  'language',
   'known',
```

Comment:
Unfortunately, this change breaks backwards compatibility. 

It is causing an e2e test in this build to fail, which involves changing the user's language locally. 
Webapp will try to call this endpoint to change the language on the server, and actually receive a 400, because no other changes are sent except language. 
The same happens when trying to update a user in the admin app. 
For offline users, it's worse as we filter which fields are editable and which are not, so a request that contains the `language` field will receive a 401.

To maintain backwards compatibility, we should make sure that requests that only try to update the language don't fail for admin users, and requests that have the `language` in the payload don't fail for offline users.